uwrt_mars_rover_controllers/uwrt_open_loop_position_controller/package.xml

```diff
@@ -0,0 +1,29 @@
+<package format="2">
+  <name>uwrt_open_loop_position_controller</name>
+  <version>0.17.0</version>
+  <description>uwrt_open_loop_position_controller</description>
+
+  <maintainer email="bence.magyar.robotics@gmail.com">Bence Magyar</maintainer>
```

Comment:
delete the maintainers list and author. or just delete the maintainers and add urself as an author and maintainer